arcgis-ios-sdk-samples/Layers/Stretch renderer/StretchRendererSettingsVC.swift

```diff
@@ -121,9 +121,7 @@ class StretchRendererSettingsVC: UITableViewController {
         guard tableView.cellForRow(at: indexPath) == stretchTypeCell else {
             return
         }
-        let labels = StretchType.allCases.map({ (type) -> String in
-            return type.label
-        })
+        let labels = StretchType.allCases.map { return $0.label }
```

Comment:
```suggestion
        let labels = StretchType.allCases.map { $0.label }
```

Reply:
Good catch - I'll tidy this up prior to merging.